libs/cli/src/main/java/org/elasticsearch/cli/Terminal.java

```diff
@@ -324,8 +325,8 @@ static class SystemTerminal extends Terminal {
                 // at the end of each character based read, so that switching to using getInputStream() returns binary data
                 // right after the last character based input (newline)
                 new InputStreamReader(System.in, Charset.defaultCharset()),
-                new PrintWriter(System.out, true),
-                ERROR_WRITER
+                new JsonPrintWriter(System.out, true),
```

Comment:
We don't want to _always_ output as json right? This should only be in the case ES is output json to stdout already, ie with docker.

Reply:
Yes, absolutely 👍 
I've added a check to look for `ELASTIC_CONTAINER`

Reply:
After testing a couple of approaches using the distro tests and going through the startup scripts more in detail, I ended up passing through a terminal format (`CLI_TERMINAL_FORMAT` / `cli.terminal.format`). This is set in `docker-entrypoint.sh` in correspondence to `ES_LOG_STYLE`.

This way we properly support `ES_LOG_STYLE=file`. But also, more importantly, this prevents issues when processing the response of `elasticsearch-keystore list`.